WhenPushComesToShove/Assets/3-Scripts/LevelSystems/EndConditions/PushDoorEndCondition.cs

```diff
@@ -72,24 +73,17 @@ private void SetActiveLights()
         lightsToTurnOff.Clear();
     }
 
-    private void OnTriggerEnter2D(Collider2D collision)
+    private void OnTriggerStay2D(Collider2D collision)
```

Comment:
Why are we doing OnTriggerStay here? is it because exit isnt being called for whatever reason?